.github/workflows/conformance.yml

```diff
@@ -27,12 +27,17 @@ env:
   GO_VERSION: "1.18"
 
 jobs:
-  run-on-kind:
+  run-on-environment:
     # Run on PR only if there is a `pr/run-conformance` label
     if: "github.event_name != 'pull_request' || contains(github.event.pull_request.labels.*.name, 'pr/run-conformance')"
     runs-on: ubuntu-latest
     strategy:
       matrix:
+        aws-region:
+          - us-west-2
```

Comment:
I would set this as a top level envar instead since I don't imagine we'll ever use multiple regions for AWS